src/plugins/shared_ux/.i18nrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "prefix": "sharedUX",
+  "paths": {
+    "sharedUX": "."
+  },
+  "translations": [
+    "translations/ja-JP.json"
```

Comment:
Hey @clintandrewhall , would you mind removing this line from `main`? Kibana crashes currently if you try to run it with the `ja-JP` locale since there is no `src/plugins/shared_ux/translations/ja-JP.json` file